tsconfig.json

```diff
@@ -1,8 +1,10 @@
 {
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
     "target": "es2019",
     "moduleResolution": "node",
     "jsx": "react",
-    "jsxFactory": "h"
+    "jsxFactory": "h",
```

Comment:
Testing locally, I don't _think_ setting isn't needed – the default setting accommodates React as standard. Is that the same with your build?

Reply:
Yeah you're right, happy to remove.